### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by rjnn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+This repository contains an extensive test suite of SQL logic
+tests. To run the bigtests on CockroachDB, perform the following
+steps:
+
+1. Clone this repository using `go get`
+2. In the CockroachDB repository, run `make testlogic`, and cancel the
+   test run once the `logictest.test` binary is built.
+3. `cd` into `pkg/sql/logictest`
+4. Run `./logictest.test -bigtest -show-sql -test.v`
```
Comment (jordanlewis) [inline README.md]: `make testlogic TESTFLAGS="-bigtest -test.v"` has the same effect. You don't want `-show-sql`. There is too much output.

**Review by jordanlewis — COMMENTED** [review]
Thanks for adding this!

File: README.md (hunk 1 above)
Comment (rjnn) [reply README.md]: Done.

**Comment by jordanlewis** [comment]
You didn't take my comment about `make testlogic`! that shortens the number of steps from 4 to 2.

**Comment by rjnn** [comment]
Bah. Sending another PR.

**Pull request merged**